app/components/developers/results_visibility_component.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module Developers
+  class ResultsVisibilityComponent < ApplicationComponent
+    renders_one :results_set, ->(records:) do
+      CardComponent.with_collection(records)
+    end
+
+    def initialize(query:, user:, paywalled: nil, size: :large, title: nil, description: nil)
+      @query = query
+      @user = user
+      @paywalled = paywalled
+      @size = size
+      @title = title
+      @description = description
+    end
+
+    def render_content?
+      pagy.page > 1 ? customer? || owner? : true
```

Comment:
This line is a bit hard to understand – can you please split it out to multiple statements?

Reply:
Yeah, I'm too lazy sometimes. My bad 😅

Reply:
Ha, all good!